front/app/components/admin/PostManager/components/ExportMenu/index.tsx

```diff
@@ -29,7 +29,7 @@ export interface Props {
   type: ManagerType;
   /** A set of ids of ideas/initiatives that are currently selected */
   selection: Set<string>;
-  selectedProject: string | undefined;
+  selectedProject?: string | undefined;
```

Comment:
Easier to not have to specify this explicitly and being able to leave it out altogether (in the proposals manager).